code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm

```diff
@@ -166,6 +170,11 @@
 /// Performs heat calculation for the freezer.
 /// We just equalize the gasmix with an object at temp = var/target_temperature and heat cap = var/heat_capacity
 /obj/machinery/atmospherics/components/unary/thermomachine/process_atmos()
+	process_atmos_checks()
+	process_atmos_heating()
+	update_parents()
```

Comment:
From my previous question, if ``process_atmos_heating()`` is the same, is there a point to changing any of this?

Reply:
From previous response, code is still very subject to change.